smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/MediatorManager.java

```diff
@@ -63,7 +63,7 @@
     private final List<AbstractMediator> mediators = new ArrayList<>();
 
     @Inject
-    @ConfigProperty(name = "smallrye.messaging.emitter.default-buffer-size", defaultValue = "127")
+    @ConfigProperty(name = "mp.messaging.emitter.default-buffer-size", defaultValue = "127")
```

Comment:
is this because we intend to move it to the spec soon and don't want to break people once it is?

Reply:
Yes, as it's a matter of weeks I believe it's ok.
We could consider this as a breaking change, but I don't believe anyone uses this property.

Reply:
ok, that's fine

Reply:
actually no, it's not fine, the documentation mentions the property, we need to support both.

Reply:
I've re-introduced the management of both properties and updated the doc.